index.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
```

Comment:
To increase your website visibility, you should add the following \<meta\> tags :
\<meta name="description" content="Description qui apparaît sous le titre dans google"\>
    \<meta name="keywords" content="WordPress"\>
    \<meta name="author" content="Nom de l'auteur"\>